### PR TITLE
Nine: some hints about upgrades

### PR DESCRIPTION
**Pull request by jaredgrubb**
Working on updating my buildmaster to nine and had some suggestions for the docs and upgrade guide.

### CODE REVIEW

File: master/docs/manual/installation/nine-upgrade.rst [hunk 1]
```diff
@@ -127,6 +127,9 @@ A few notes on changes to the configuration of these reporters:
 
 * :bb:reporter:`GitHubStatus` now accepts a ``context`` parameter to be passed to the GitHub Status API.
 
+* :py:class:`~buildbot.status.builder.Results` and the constants :py:class:`~buildbot.status.results.SUCCESS` should be imported as ____#FIXME!!!_____ instead.
+
```
Comment (jaredgrubb) [inline master/docs/manual/installation/nine-upgrade.rst]: If the goal is to remove use of "buildbot.status", what is the replacement for these? These seem common enough that maybe they should just be split to a special file?

Comment (tardyp) [inline master/docs/manual/installation/nine-upgrade.rst]: imho should go in buildbot.process.results

**Comment by tardyp** [comment]
thanks for those contribs! :+1:

**Comment by sa2ajj** [comment]
```
[FAIL]
Traceback (most recent call last):
  File "/home/travis/build/buildbot/buildbot/master/buildbot/test/unit/test_config.py", line 817, in test_load_status_not_status_rec
    self.assertConfigError(self.errors, "must be a list of")
  File "/home/travis/build/buildbot/buildbot/master/buildbot/test/util/config.py", line 31, in assertConfigError
    "expected: %s" % (curr_error, substr_or_re))
twisted.trial.unittest.FailTest: non-matching error: c['status'] contains an object that is not a status receiver (type <type 'str'>), expected: must be a list of

buildbot.test.unit.test_config.MasterConfig_loaders.test_load_status_not_status_rec
```

**Comment by jaredgrubb** [comment]
Fixed the unit test failure; fixed a docs issue; will do the 'buildbot.process.results" piece in another patch.

**Comment by tardyp** [comment]
:+1:

File: master/docs/manual/installation/nine-upgrade.rst [hunk 2]
```diff
@@ -152,6 +153,13 @@ The following fields are identifiers:
 * builder name (20-character)
 * step name (50-character)
 
+Other Config Settings
+---------------------
+
+The default master.cfg file contains some new changes, which you should look over:
+* ``c['protocols'] = {'pb': {'port': 9989}}`` (the default port used by the buildslaves)
+* Waterfall View: requires installation (``pip install buildbot-waterfall-view``) and configuration (``c['www'] = { ..., 'plugins': {'waterfall_view': {} }``).
+
```
Comment (djmitche) [inline master/docs/manual/installation/nine-upgrade.rst]: Good catches both

**Pull request merged**